Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -15,10 +15,27 @@
 
 @implementation BPPacker
 
++ (NSMutableSet *)fullyExpandedTests:(NSArray *)configTestCases fromTestFile:(BPXCTestFile *)xctFile {
```

Comment:
This has been addressed.  There is now a function called `normalizedBPConfiguration`, which fully expands the test suites in the config.  This meant we could revert the changes to `packTests`, instead it just calls `normalizedBPConfiguration`.